src/Page.ts

```diff
@@ -920,6 +920,36 @@ export class Page extends EventEmitter {
     }
   }
 
+  async emulateVisionDeficiency(
+    type?:
+      | 'none'
+      | 'achromatopsia'
+      | 'blurredVision'
+      | 'deuteranopia'
+      | 'protanopia'
+      | 'tritanopia'
+  ): Promise<void> {
+    const visionDeficiencies = new Set([
+      'none',
+      'achromatopsia',
+      'blurredVision',
+      'deuteranopia',
+      'protanopia',
+      'tritanopia',
+    ]);
```

Comment:
I think we could define the vision deficiency type as an enum here and take advantage of their runtime behaviour to avoid duplicating this list, maybe something like this: https://www.typescriptlang.org/docs/handbook/enums.html#enums-at-compile-time

So we define the enum:

```
enum VisionDeficiency {
  none,
  achromatopsia,
  ... 
} 
```

And then create a set using it because it will actual exist at runtime:  

```
const visionDefs = new Set(Object.keys(VisionDeficiency))
```

Or similar. It might not be exactly this but it will clear up some duplication here. Does `protocol.d.s` generate any types for this set that we can use?